### ISSUE TITLE
Jquery 1.9.1 Incompatibility

### ISSUE THREAD

**Issue by Devolv** [pisi/Reel#174]
Running the latest Jquery 1.9.1 results in the following Error:

TypeError: k.browser is undefined

When switching back to legacy Jquery 1.8.3, Reel works again.

**Comment by pisi** [comment]
In other words: time for a patch.

Thanks for the notice!

**Comment by pisi** [comment]
Code on the [development branch] is jQuery 1.9 compatible (in fact it works well with 2.0 beta too!) and is to be released as Reel 1.2.1 within few days.

**Comment by pisi** [comment]
@dtrunk90, thanks very much for sharing your fix, which I noticed just now. Kudos to you, sir! The fix indeed does make the trick. I applied a similar solution and though I realize the check there is to specifically catch IE, I came up with a stack of regular expressions (`$.reel.re.ua`) to identify the browser on my own. It basically is an optimized version of what's in the jQuery Migrate poly-fill plugin (whose full logic I employed in tests, which support this feature). However, thank you very much for your effort, Danny :)
